src/delegation/delegationTransactionsFactory.ts

```diff
@@ -20,7 +21,7 @@ interface IConfig {
     additionalGasLimitForDelegationOperations: bigint;
 }
 
-interface IValidatorPublicKey {
+export interface IValidatorPublicKey {
```

Comment:
Should not be exported, this is an internal interface. Can be handled as the other re-defined interfaces.